src/jit/compiler.hpp

```diff
@@ -4280,6 +4280,42 @@ bool                Compiler::compIsForInlining()
     return (impInlineInfo != NULL);
 }
 
+//------------------------------------------------------------------------------
+// compIsUnixAmd64ABI : Returns true if the compiler is for compiling UnixAmd64 ABI.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    True if the compiler is for compiling UnixAmd64ABI. False otherwise.
+inline
+bool Compiler::compIsUnixAmd64ABI()
```

Comment:
I thinkthat this reads better if it is called:   isUnixAmd6ABI

Please add an additional API called:

bool isWindowsAmd64ABI()